packages/ckeditor5-mention/tests/manual/mention-custom-dropdownlimit.md

```diff
@@ -0,0 +1,13 @@
+## Mention
+
+The mention configuration with a custom `config.mention.dropdownLimit` configuration and a static list of autocomplete feed.
+
+### Configuration
+
+Type "@" to display the list of 20 mentions out of 21 available or type "#" to see shorter list.
+
+### Interaction
+
+- Move arrows up/down to select an item.
+- Use <kbd>enter</kbd> or <kbd>tab</kbd> to insert a mention into the documentation.
```

Comment:
I know it says "documentation" in other manual tests but I think it's not worth to propagate a typo, let's change it to the "document".

Reply:
I've removed the typo in all manual tests: f2826ae.